### ISSUE TITLE
Source browser module

### ISSUE THREAD

**Issue by daveaglick** [statiqdev/Statiq#121]
As a follow-on to #70, the code analysis data combined with the input source file documents should be used to create "source browser" pages similar to what the Roslyn team is currently doing. All the data is there, this will mainly involve inserting links for the generated symbol documents into the source file documents at the appropriate places.

**Comment by jairbubbles** [comment]
It's indeed cool. Being able to navigate through files by clicking on symbols is a killer feature.

**Comment by LennardF1989** [comment]
Is this on the road-map any time soon? Amount of "want" is really high!

**Comment by daveaglick** [comment]
@LennardF1989 I agree, also really want this. Right now though, everything has a back seat to the .NET Core porting effort. Thankfully that’s going pretty well, so I may be able to get to this late spring or summer.

**Comment by LennardF1989** [comment]
@daveaglick If you have any pointers for me on how you were thinking to achieve this, I can spare some time. It doesn't seem too hard to include a simple highlight library, and inject another page per class (with anchors), just no idea in which kind of pipeline you would want this to happen.
